app/products/calls/connection/connection.ts

```diff
@@ -212,6 +213,10 @@ export async function newConnection(serverUrl: string, channelID: string, closeC
                 disconnect();
             }
         });
+
+        peer.on('connect', () => {
+            setCurrentCallConnected();
+        });
```

Comment:
Not opposed to this but just pointing out that we'd be slightly diverging from web client's behaviour that takes the websocket event as the confirmation of a connection:

https://github.com/mattermost/mattermost-plugin-calls/blob/4eb1207bf845137503fab38ac2aae1cb74694717/webapp/src/reducers.ts#L182-L185

But probably we can review all this when we finally add support for multiple sessions per user in the same call.

Reply:
@streamer45 Right -- we could move to using that as the trigger (the CurrentCall basically acts as the callsClient for this purpose)... Let's do that, thanks!